releases/firmware-btc-v9.3.1/assertion.txt

```diff
@@ -0,0 +1,8 @@
+By signing this file, the signer confirms that the firmware binary built from:
+
+git tag firmware-btc-only/v9.3.0
```

Comment:
typo, 9.3.0 should be 9.3.1

Reply:
:see_no_evil: I looked only at hash values.